server/start.go

```
@@ -272,7 +273,14 @@ func startNode(
 
 	nodeKey := &key.NodeKey{PrivKey: signingKey, PubKey: signingKey.GetPublic()}
 
-	rollkitcfg, err := config.Load(rootCmd)
+	cmd := &cobra.Command{}
+	flags := &pflag.FlagSet{}
+	for key, value := range srvCtx.Viper.AllSettings() {
+		flags.Set(key, fmt.Sprintf("%v", value))
```

golangci-lint (1.64.8)

Error return value of `flags.Set` is not checked

(errcheck)